docs/conf.py

```diff
@@ -26,6 +26,9 @@
 
 import openforcefield
 
+from recommonmark.transform import AutoStructify
```

Comment:
This was not pulled down for me with `conda env create --name openff-docs --file environment.yml`